boards/arm/nrf5340dk_nrf5340/Kconfig.board

```diff
@@ -11,8 +11,18 @@ config BOARD_NRF5340PDK_NRF5340_CPUAPP
 config BOARD_NRF5340PDK_NRF5340_CPUAPPNS
 	bool "nRF5340 PDK nRF5340 Application MCU non-secure"
 
+config BOARD_NRF5340DK_NRF5340_CPUAPP
```

Comment:
Could we provide Kconfig alias that does not distinguish between DK and PDK version of the board?

It could simplify a lot of if-endifs in this PR.